Makefile

```diff
@@ -35,16 +35,18 @@ all-srcs:
 .PHONY: fmt-vet-lint-test
 fmt-vet-lint-test: fmt vet lint test
 
-.PHONY: perf-test
-perf-test: otelsvc
+.PHONY: e2e-test
+e2e-test: otelsvc
 	$(MAKE) -C testbed runtests
 
 .PHONY: test
 test:
 	$(GOTEST) $(GOTEST_OPT) $(ALL_PKGS)
 
 .PHONY: travis-ci
-travis-ci: fmt vet lint test-with-cover
+travis-ci: fmt vet lint test-with-cover otelsvc
```

Comment:
Do we want to run E2E test on every commit? Maybe only run it once per day?

Reply:
We only have a small number of E2E tests and they are quick to run. I think we should run them on every build for now. We can decide to run them less frequently when they start taking a lot of time.